### PR TITLE
DOCS-2950 Add static IP configuration instructions to NextCloud article

### PR DESCRIPTION
**Pull request by tonyriv3**
TrueNAS users are reporting difficulty with accessing the NextCloud UI when the plugin is configured with a static IP address and not NAT. Review https://www.truenas.com/community/threads/nextcloud-doesnt-recognize-authorized-users.97211/#post-671279 and update https://www.truenas.com/docs/core/solutions/integrations/nextcloud/  to document configuring the plugin with a static IP address and then accessing the NextCloud UI.

### CODE REVIEW

File: content/CORE/Solutions/Integrations/Nextcloud.md [hunk 1]
```diff
@@ -3,18 +3,14 @@ title: "Nextcloud"
...
-### Plugin Installation ###
-
-{{< tabs "TrueNAS Options" >}}
-{{< tab "TrueNAS CORE" >}}
 ### Plugins Catalog
 
 {{< expand "Before getting started..." "v" >}}
-* A [data pool]({{< relref "PoolCreate.md" >}}) must be available for plugin storage.
+* You must have a [data pool]({{< relref "PoolCreate.md" >}}) available for plugin storage.
 * The system must be connected to the internet.
```
Comment (micjohnson777) [inline content/CORE/Solutions/Integrations/Nextcloud.md:14]:
```suggestion for parallel construction maybe this
* You must connect the system to the internet.
```

File: content/CORE/Solutions/Integrations/Nextcloud.md [hunk 2]
```diff
@@ -3,18 +3,14 @@ title: "Nextcloud"
...
 * Go to **Network > Interfaces**, edit the intended plugin interface, and set *Disable Hardware Offloading*.
```
Comment (micjohnson777) [inline content/CORE/Solutions/Integrations/Nextcloud.md:15]:
```suggestion maybe remove this from the bullet list as it isn't a 'requirement' like the other two are characterized as, make it a separate action after the others...
Go to **Network > Interfaces**, edit the intended plugin interface, and set **Disable Hardware Offloading**.
```

File: content/CORE/Solutions/Integrations/Nextcloud.md [hunk 3]
```diff
@@ -25,105 +21,80 @@ To see the plugin catalog, go to the **Plugins** screen.
...
+* Open the **Shell** and `ping` an Internet address. 
+The output confirms the system is connected to the Internet.
 {{< /expand >}}
 
-Plugins are organized into two **Collections**:
+TrueNAS organizes plugins into two **Collections**:
 
 * [iXsystems](https://www.ixsystems.com/) maintained plugins
-* Open Source plugins created and maintained by the TrueNAS community.
+* Open Source plugins created and maintained by TrueNAS users.
```
Comment (micjohnson777) [inline content/CORE/Solutions/Integrations/Nextcloud.md:34]:
```suggestion
* Open source plugins created and maintained by TrueNAS users.
```

File: content/CORE/Solutions/Integrations/Nextcloud.md [hunk 4]
```diff
@@ -25,105 +21,80 @@ To see the plugin catalog, go to the **Plugins** screen.
...
 
 ![NextcloudInstall](/images/CORE/12.0/SolutionsIntegrationsNextcloudInstall.png "Nextcloud Install")
 
-For *Plugins / Add* choose a *Jail Name* and click *SAVE*
+Type a **Jail Name** and click **SAVE**.
 
 ![NextcloudJailName](/images/CORE/12.0/SolutionsIntegrationsNextcloudJailName.png "Nextcloud Jail Name")
 
-Once Nextcloud has installed successfully, you can then begin to manage your instance of the plugin.  
+Once Nextcloud installs successfully, you can manage your instance of the plugin.
```
Comment (micjohnson777) [inline content/CORE/Solutions/Integrations/Nextcloud.md:51]:
```suggestion you can use after or when here as a better choice
After Nextcloud installs successfully, you can manage your instance of the plugin.  
```

File: content/CORE/Solutions/Integrations/Nextcloud.md [hunk 5]
```diff
@@ -25,105 +21,80 @@ To see the plugin catalog, go to the **Plugins** screen.
...
 
 ![NextcloudJailName](/images/CORE/12.0/SolutionsIntegrationsNextcloudJailName.png "Nextcloud Jail Name")
 
-Once Nextcloud has installed successfully, you can then begin to manage your instance of the plugin.  
+Once Nextcloud installs successfully, you can manage your instance of the plugin.  
 
 ![NextcloudPostInstall](/images/CORE/12.0/SolutionsIntegrationsNextcloudPostInstall.png "Nextcloud Post Install")
 
-Select *POST INSTALL NOTES* to obtain your *Nextcloud Admin User* and *Nextcloud Admin Password* information.  Select *MANAGE* and you will be directed to the Nextcloud login page within your browser.
+Click **POST INSTALL NOTES** to obtain your *Nextcloud Admin User* and *Nextcloud Admin Password* information.
```
Comment (micjohnson777) [inline content/CORE/Solutions/Integrations/Nextcloud.md:55]: if you are specifically talking about the UI elements as UI elements they are bold, if only talking about them in general  as an item or value for the UI fields then no emphasis is necessary. 
```suggestion
Click **POST INSTALL NOTES** to obtain your Nextcloud admin user and  Nextcloud admin password information. 
```

File: content/CORE/Solutions/Integrations/Nextcloud.md [hunk 6]
```diff
@@ -25,105 +21,80 @@ To see the plugin catalog, go to the **Plugins** screen.
...
-Official containers are pre-configured to only require a name during deployment.
-{{< /hint >}}
+Type a **Jail Name**, then disable the **NAT** checkbox and enter an available IP in the **IPv4 Address** field. 
+Select an **IPv4 Netmask** (iX recommends 24), then click **SAVE**.
 
-Go to **Apps** and click the *INSTALL* button.  Enter an *Application Name* and click *Next*.
+![NextcloudStatic1](/images/CORE/NextcloudStatic1.png "Nextcloud Static Setup")
 
-![SCALENextcloudInstall](/images/SCALE/SCALENextcloudInstall.png "SCALE Nextcloud Install")
+Once Nextcloud installs, you must add your Nextcloud IP to your Nextcloud jail trusted domains.
```
Comment (micjohnson777) [inline content/CORE/Solutions/Integrations/Nextcloud.md:75]:
```suggestion
After Nextcloud installs, you must add your Nextcloud IP to your Nextcloud jail trusted domains.
```

File: content/CORE/Solutions/Integrations/Nextcloud.md [hunk 7]
```diff
@@ -25,105 +21,80 @@ To see the plugin catalog, go to the **Plugins** screen.
...
 
-![SCALENextcloudConfirmOptions](/images/SCALE/SCALENextcloudConfirmOptions.png "SCALE Nextcloud Confirm Options")
+![NextcloudTrustedDomain](/images/CORE/NextcloudTrustedDomain.png "Nextcloud Add Trusted Domain")
 
-The Nextcloud application will now be present on the *Installed Applications* page.  Select *Portal* and you will be directed to the *Nextcloud* login page within your browser.
+Type <kdb>CTRL+C</kdb> to close the editor, then type **exit** to close the config file.
 
-![SCALENextcloudLogin](/images/SCALE/SCALENextcloudLogin.png "SCALE Nextcloud Login")
+Go back to **Plugins** and expand your Nextcloud instance. 
+Click **POST INSTALL NOTES** to obtain your *Nextcloud Admin User* and *Nextcloud Admin Password* information. Click **MANAGE** to access the Nextcloud login page within your browser.
```
Comment (micjohnson777) [inline content/CORE/Solutions/Integrations/Nextcloud.md:88]: Same as above, if not specifically talking about the UI field and only the value items for the UI field, no emphasis required
```suggestion
Click **POST INSTALL NOTES** to obtain your Nextcloud admin user and Nextcloud admin password information. Click **MANAGE** to access the Nextcloud login page within your browser.
```

File: content/CORE/Solutions/Integrations/Nextcloud.md [hunk 8]
```diff
@@ -25,105 +21,80 @@ To see the plugin catalog, go to the **Plugins** screen.
...
 
-Enter the credentials specified during installation and click *Log in*.  You will be directed to the *Nextcloud Hub*.
+![NextcloudLogin](/images/CORE/12.0/SolutionsIntegrationsNextcloudLogin.png "Nextcloud Login")
 
-![NextcloudHub](/images/CORE/12.0/SolutionsIntegrationsNextcloudLogin.png "Nextcloud Hub")
+Enter the credentials from **POST INSTALL NOTES** and click **Log in**.  You will be directed to the Nextcloud Hub.
```
Comment (micjohnson777) [inline content/CORE/Solutions/Integrations/Nextcloud.md:92]:
```suggestion to remove 'will'
Enter the credentials from **POST INSTALL NOTES** and click **Log in**.  You are directed to the Nextcloud Hub.
```

File: content/SCALE/Apps/NextcloudSCALE.md [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+
+### Plugin Installation
+
+{{< hint info >}}
+**Official Applications**
+ 
+Official containers are pre-configured to only require a name during deployment.
+{{< /hint >}}
+
+Go to **Apps** and click the *INSTALL* button.  Enter an *Application Name* and click *Next*.
```
Comment (micjohnson777) [inline content/SCALE/Apps/NextcloudSCALE.md:18]:
```suggestion
Go to **Apps** and click the **INSTALL* button.  Enter an **Application Name** and click **Next**.
```

File: content/SCALE/Apps/NextcloudSCALE.md [hunk 10]
```diff
@@ -0,0 +1,42 @@
...
+
+![SCALENextcloudInstall](/images/SCALE/SCALENextcloudInstall.png "SCALE Nextcloud Install")
+
+For *Nextcloud Configuration* the *Username* defaults to *admin* and *Password* defaults to *changeme*.  It is recommended that you change both of these fields to entries specific to you as the user.  Ensure you employ a password that is both strong and can be remembered.
```
Comment (micjohnson777) [inline content/SCALE/Apps/NextcloudSCALE.md:22]:
```suggestion
For **Nextcloud Configuration** the **Username** value defaults to **admin** and the **Password** value defaults to **changeme**.  We recommend that you change both of these fields to entries specific to you as the user.  Ensure you employ a password that is both strong and memberable.
```

File: content/SCALE/Apps/NextcloudSCALE.md [hunk 11]
```diff
@@ -0,0 +1,42 @@
...
+
+![SCALENextcloudConfiguration](/images/SCALE/SCALENextcloudConfiguration.png "SCALE Nextcloud Configuration")
+
+For *Storage*, *Scaling/Upgrading Policy* and *Advanced DNS Settings* the defaults can be used for basic installation by clicking *Next* or bypassing and going directly to *Confirm Options*.  To *Confirm Options*, review the information presented and ensure that it is all correct.  Click *Submit*.
```
Comment (micjohnson777) [inline content/SCALE/Apps/NextcloudSCALE.md:26]:
```suggestion also not certain if the last sentence is talking specifically about a UI element or screen name or the action of confirming options, if it is specifically the UI element then use bold, of talking about the action of confirm options then reword and not emphasis and use sentence capitalization i.e. "to confirm options, ..."
For the **Storage**, **Scaling/Upgrading Policy** and **Advanced DNS Settings** you can use the defaults for basic installation by clicking **Next** or bypassing and going directly to **Confirm Options**.  To *Confirm Options*, review the information presented and ensure that it is all correct.  Click *Submit*.
```

File: content/SCALE/Apps/NextcloudSCALE.md [hunk 12]
```diff
@@ -0,0 +1,42 @@
...
+
+![SCALENextcloudConfirmOptions](/images/SCALE/SCALENextcloudConfirmOptions.png "SCALE Nextcloud Confirm Options")
+
+The Nextcloud application will now be present on the *Installed Applications* page.  Select *Portal* and you will be directed to the *Nextcloud* login page within your browser.
```
Comment (micjohnson777) [inline content/SCALE/Apps/NextcloudSCALE.md:30]:
```suggestion
The Nextcloud application is now present on the **Installed Applications** page.  Select **Portal** and you are directed to the **Nextcloud** login page within your browser.
```

File: content/SCALE/Apps/NextcloudSCALE.md [hunk 13]
```diff
@@ -0,0 +1,42 @@
...
+
+![SCALENextcloudLogin](/images/SCALE/SCALENextcloudLogin.png "SCALE Nextcloud Login")
+
+Enter the credentials specified during installation and click *Log in*.  You will be directed to the *Nextcloud Hub*.
```
Comment (micjohnson777) [inline content/SCALE/Apps/NextcloudSCALE.md:34]:
```suggestion
Enter the credentials specified during installation and click *Log in**.  You are directed to the **Nextcloud Hub**.
```

**Comment by micjohnson777** [comment]
Changes future tense 'will' to present tense.
It's tricky when we mention UI elements in running text and knowing when to use emphasis, like bold for the UI element or no emphasis when we are talking about the values for the UI element or action they represent. Sometimes just rewording a bit helps make it more clear and the emphasis to use more clear as well. I'll make sure I include this in the style guide with more examples. :-)

**Comment by tonyriv3** [comment]
@micjohnson777 
I committed all your suggestions.

**Review by Mrt134 — APPROVED** [review]

**Comment by Mrt134** [comment]
Previous build test passed 100% - current failure is a hiccup from surge.sh. Approved for merge.

**Pull request merged**